bindings/python/iota_sdk/types/token_scheme.py

```diff
@@ -22,20 +22,5 @@ class SimpleTokenScheme:
     maximum_supply: HexStr
     type: int = field(default_factory=lambda: 0, init=False)
 
-    @staticmethod
-    def _to_dict_custom(config):
```

Comment:
Don't we want to keep this if we want to use `int` for the token amounts later?

Reply:
I guess custom encoder/decoder would be better and we still have the git history for this, created a new issue https://github.com/iotaledger/iota-sdk/issues/1494